napari_clusters_plotter/_plotter.py

```diff
@@ -9,6 +9,7 @@
 from qtpy.QtCore import Qt
 from qtpy.QtGui import QGuiApplication, QIcon
 from qtpy.QtWidgets import QComboBox, QHBoxLayout, QLabel, QVBoxLayout, QWidget
+from vispy.color import Color
```

Comment:
Local imports and smarter than global imports because the Python code get's loaded faster when starting the plugin...